pkg/network/apis/network/validation/validation.go

```diff
@@ -37,13 +36,6 @@ func validateIPv4(ip string) (net.IP, error) {
 	return bytes, nil
 }
 
-var defaultClusterNetwork *networkapi.ClusterNetwork
-
-// SetDefaultClusterNetwork sets the expected value of the default ClusterNetwork record
-func SetDefaultClusterNetwork(cn networkapi.ClusterNetwork) {
-	defaultClusterNetwork = &cn
```

Comment:
So this was really weird.  This is a method that sets a global that was only called from a package that ran as part of the controller manager.

Thinking back, perhaps that import is accidentally called from the server.....  I'll double check

Reply:
yeah, only called when you start the SDN controller which happens in a different process than the one that does the validation.